File: cinema.log.server/cinema.log.server/Models/Entities/Review.cs
using System.ComponentModel.DataAnnotations;

namespace cinema.log.server.Models.Entities;

public class Review
{
    public Guid ReviewId { get; set; }
    
    [MaxLength(3000)]
    public string? Content { get; set; }
    
    [Required]
    public DateTime Date { get; set; }
    
    [Required]
    public float Rating { get; set; }
    
    [Required]
    public required Guid FilmId { get; set; }
    
    [Required]
    public required Guid UserId { get; set; }
}

File: cinema.log.server.golang/internal/domain/review.go
package domain

import (
	"time"

	"github.com/google/uuid"
)

type Review struct {
	ID      uuid.UUID `json:"id"`
	Content string    `json:"title"`
	Date    time.Time `json:"date"`
	Rating  float32   `json:"rating"`
	FilmId  uuid.UUID `json:"filmId"`
	UserId  uuid.UUID `json:"userId"`
}
